build-static-bins.sh

```diff
@@ -4,7 +4,7 @@
 # and with logging enabled (controlled by RUST_LOG env var)
 #
 # You need the static version of glibc installed for this to work.
-# On Fedora/RHEL that's: glibc-static.
+# On Fedora/RHEL that's: sudo dnf install -y rust cargo glibc-static
```

Comment:
I do not like the asymmetry between Ubuntu and Fedora now. One has the full command and one just has a package name. And one is rightfully just about glibc while the other lists all dependencies.